sites/public/src/pages/listing/ext/[id].tsx

```diff
@@ -91,7 +91,7 @@ export async function getServerSideProps(context: {
   return {
     props: {
       listing: response.data,
-      jurisdiction: await getBloomJurisdictionById(response.data.jurisdiction.id),
+      jurisdiction: response.data.jurisdiction,
```

Comment:
I haven't pulled down the branch to test, but is this enough data? the jurisdiction object from /jurisdictions/byName (https://api.housingbayarea.org/jurisdictions/byName/Alameda) is significantly bigger than the jurisdiction object within the listing object (https://api.housingbayarea.org/listings/38b34d9d-591b-4334-a623-4776e2bbdbee -- it only has 3 fields)

My guess is that we'll need to keep the etBloomJurisdictionById jurisdiction fetching logic in order to build this component

Reply:
There are only two props on jurisdiction that are used in `ListingView`: `enableUtilitiesIncluded` and `enableAccessibilityFeatures`.  Per Morgan, these are feature flags that are being removed, so it's safe to use a version of jurisdiction that doesn't have these.  At some point jurisdiction can be removed from `ListingView` entirely, but I didn't want to go that far with it.

Reply:
got it, thanks